README.md

```diff
@@ -176,11 +191,231 @@ Example (the following are equivalent):
 ]
 ```
 
+### List Secrets
+
+```bash
+spinup list secrets funSpace sensitive
+spinup list secrets -s 128 -s 136
+```
+
+```json
+[
+  {
+    "name": "MySecretString",
+    "description": "beer",
+    "space_id": "128"
+  }
+]
+```
+
+### List Images
+
+```bash
+spinup list images funSpace sensitive
+spinup list images -s 128 -s 136
+```
+
+```json
+[
+  {
+    "architecture": "x86_64",
+    "created_at": "2020/05/05 15:25:06",
+    "created_by": "s_service_user",
+    "description": "My Perfect Image. So nice.",
+    "id": "ami-008f837464770f445",
+    "name": "Perfect",
+    "server_name": "spintst-123456.spinup.yale.edu",
+    "state": "available",
+    "status": "created",
+    "volumes": {
+      "/dev/sda1": {
+        "delete_on_termination": true,
+        "encrypted": false,
+        "snapshot_id": "snap-00001111122223334444",
+        "volume_size": 30,
+        "volume_type": "gp2"
+      }
+    },
+    "offering_id": "18",
+    "offering_name": "CentOS 7",
+    "space_id": "128"
+  }
+]
+```
+
 ## Get Commands
 
 The `get` subcommands allow you to get detailed information about a resource.
 
-### TODO
+### Get Container Summary
+
+```bash
+spinup get container 2120
+```
+
+```json
+{
+  "id": "2120",
+  "name": "spintst-000848-testService",
+  "status": "created",
+  "type": "Container Service",
+  "flavor": "fargate",
+  "security": "low",
+  "space_id": "128",
+  "beta": false,
+  "size": "nano.512MB",
+  "tryit": false,
+  "state": "ACTIVE"
+}
+```
+
+### Get Container Details
+
+```bash
+spinup get container 2120 -d
+```
+
+```json
+{
+  "id": "2120",
+  "name": "spintst-000848-testService",
+  "status": "created",
+  "type": "Container Service",
+  "flavor": "fargate",
+  "security": "low",
+  "space_id": "128",
+  "beta": false,
+  "size": "nano.512MB",
+  "tryit": false,
+  "state": "ACTIVE",
+  "details": {
+    "desiredCount": 1,
+    "endpoint": "spintst-000848-testService.svc.spinup.yale.edu",
+    "pendingCount": 1,
+    "runningCount": 0,
+    "containers": [
+      {
+        "auth": false,
+        "image": "yalespinup/nginxproxy",
+        "name": "nginxproxy",
+        "env": {
+          "BACKEND_URL": "http://127.0.0.1:8080"
+        },
+        "portMappings": [
+          "8443/tcp"
+        ],
+        "secrets": {}
+      },
+      {
+        "auth": true,
+        "image": "yalespinup/testapi",
+        "name": "api",
+        "env": {
+          "FOOFOOFOO": "kJBDGKLBEGLKWBGLsndlkFNFGLKEN",
+          "dKJGBLSGB": "LDNGLWK"
+        },
+        "portMappings": [
+          "8080/tcp"
+        ],
+        "secrets": {
+          "DERPDERP": "MySecretString"
```

Comment:
Are secrets going to be displayed?

Reply:
just the name - the API doesn't return the values